pages/api/auction/createOrEditAuction.ts

```diff
@@ -8,11 +8,13 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
   const { signature, auction } = req.body
   const owner = await crabV2Contract.owner()
 
-  try {
-    const isOwner = verifyMessage(KING_CRAB, signature, owner)
-    if (!isOwner) return res.status(401).json({ message: 'Not owner' })
-  } catch (e) {
-    return res.status(401).json({ message: "Signature can't be verified" })
+  if (process.env.environment != 'dev') {
```

Comment:
let's use process.env.NODE_ENV instead. It's already provided by next

Reply:
Done